PythonForDelphi/Components/Sources/Core/PythonEngine.pas

```diff
@@ -5349,12 +5351,12 @@ function TPythonEngine.Run_CommandAsObjectWithDict(const command : AnsiString; m
 
 procedure TPythonEngine.ExecStrings( strings : TStrings );
 begin
-  Py_XDecRef( Run_CommandAsObject( EncodeString(strings.Text) , file_input ) );
+  Py_XDecRef( Run_CommandAsObject( EncodeString(strings.Text) , file_input ) );
 end;

```

Comment:
What is the difference here?

Reply:
I don't know. Probably spaces. I was also surprised.  I did not find differences.